### DISCUSSION TITLE
Why are docs for the API not public?

### DISCUSSION THREAD

**Discussion by eavilaes** [databricks/koalas#2195] [Q&A] [answered]
Hi everyone! I've been checking Koalas docs and I'd like to contribute on few things I've found to be incorrect, but this is not letting me help 😥:
https://github.com/databricks/koalas/blob/e971d6f37ede45297bbf9d509ae2a7b51717f322/docs/.gitignore#L1

Any hints on why this part of the docs is not public to contribute?
Thanks!🙌🏼

**Comment by itholic** [comment, marked as answer]
Thanks for your contribution!!

Koalas use [sphinx](https://www.sphinx-doc.org/en/master/) to build the docs.

So if you want to fix some API documents, you can fix the docstring in source codes.

For example, if you want to fix `DataFrame.astype` API documents, fix the below docstring:

https://github.com/databricks/koalas/blob/e971d6f37ede45297bbf9d509ae2a7b51717f322/databricks/koalas/frame.py#L8185-L8238

After fix the docstring, you can check after building documents: https://koalas.readthedocs.io/en/latest/development/contributing.html#building-documentation

Btw, Koalas is ported into PySpark from Apache Spark 3.2.

So, if you're also interested in contributing to Apache Spark, you can contribute to PySpark side after the 3.2 release.

**Reply by eavilaes** [reply]
Thank you! I managed to make it work :)

I appreciate your time! 😄

**Reply by itholic** [reply]
No problem! :-)
